static/styles.css

```diff
@@ -30,15 +30,19 @@ a,
 @media only screen and (min-width: 750px) {
   .grid-container {
     display: grid;
-    grid-template-columns: 80% 20%;
+    grid-template-columns: 33.33% 33.33% 33.33%;
     grid-template-rows: 1fr;
-    gap: 0px 0px;
-    grid-template-areas: "intro-text refresh";
+    grid-gap: 0px 10px;
+    grid-template-areas: "intro-text refresh2 refresh";
```

Comment:
I'm guessing this will cause responsiveness issues on mobile; did you try this on a tiny screen?